IPython/html/static/style/style.min.css

```diff
@@ -8925,6 +8925,7 @@ ul#new-menu {
   text-rendering: auto;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  transform: translate(0, 0);
```

Comment:
@minrk  - Ah it's a change introduced by https://github.com/ipython/ipython-components/pull/53 .  It looks like this is the first CSS recompile since that was merged.